site/src/pages/api/rewrites/sandboxed-block-demo.api.ts

```diff
@@ -142,8 +143,11 @@ const handler: NextApiHandler = async (req, res) => {
       
           const entryPoint = blockType.entryPoint.toLocaleLowerCase();
           
-          const rawBlockSource = source;
-          const blockExport = entryPoint === "html" ? rawBlockSource : findBlockExport(loadCjsFromSource(rawBlockSource));
+          let blockExport = findBlockExport(loadCjsFromSource(source));
```

Comment:
I hadn't clocked that the HTML block the template produces is a JS file that exports an HTML string, which is at odds with [how the spec describes HTML blocks as having an HTML entry point](https://blockprotocol.org/docs/spec/core-specification#types-of-block) – I think there are two options:

1. Have HTML blocks be either an HTML file OR a JS file that exports HTML (not crazy about this as it adds more branching, depends on what you think about 2)

2. Update the block template so that what is actually produced is an HTML file with the result of the current `index.js`, not `index.js` itself. 

What do you reckon?